bleachbit/Special.py

```diff
@@ -42,10 +42,10 @@ def __shred_sqlite_char_columns(table, cols=None, where=""):
     cmd = ""
     if cols and options.get('shred'):
         cmd += "update %s set %s %s;" % \
-            (table, ",".join(["%s = randomblob(length(%s))" % (col, col)
```

Comment:
This file has a merge conflict with commit c43077dcf55d79f20706da66d1aaacd6e800ee87